static/js/channel.js

```diff
@@ -51,8 +54,40 @@ function call(args, idempotent) {
             blueslip.log(`Ignoring ${args.type} ${args.url} error response while reloading`);
             return;
         }
-
-        if (xhr.status === 403) {
+        console.log(setup.password_change_in_progress);
+        console.log(setup.xhr_password_changes.get(xhr));
+        console.log(setup.xhr_password_changes);
+        console.log(xhr);
+        console.log(setup.password_changes);
+        if (
+            setup.password_change_in_progress ||
+            setup.xhr_password_changes.get(xhr) !== setup.password_changes
+        ) {
+            // The backend for handling password change API requests
+            // will replace the user's session; this results in a
+            // brief race where any API request will fail with a 401
+            // error after the old session is deactivated but before
+            // the new one has been propagated to the browser.  So we
+            // skip our normal HTTP 401 error handling if we're in the
+            // process of executing a password change.
+            return;
```

Comment:
I'm not sure we want this `return` logic to occur here; I think it should only be scoped in the 401 code block.